### DISCUSSION TITLE
Show limits in the date drop-down based on data source (at least for the Grafana-supported ones)

### DISCUSSION THREAD

**Discussion by tab0r** [grafana/grafana#63486] [Explore]
Since many datasources have defined query limits accessible via API, can we utilize this to populate the date selector with only valid options?

**Comment by tab0r** [comment]
For example, the GrafanaCloud Loki datasource only allows 28 days of query range. Since 30 days is included in the drop down, it's really easy to hit the query range too long error.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). We are migrating this discussion to an issue and closing the discussion. The issue is #82823. Feel free to continue the discussion around this there. Thank you!
